js/http.js

```diff
@@ -123,7 +123,8 @@ var ERMrest = (function (module) {
                         deferred.resolve(response);
                     },
                     function(response) {
-                        if (response.status in _retriable_error_codes && count < max_retries) {
+                        var status = response.status || response.statusCode;
```

Comment:
@chiragsanghvi I'm curious. I haven't seen `statusCode` as a property on the `response` from angular. Is that something that Angular used to do or still does or that other libraries use that we should handle in general?